promql/engine.go

```diff
@@ -584,6 +586,11 @@ func (ng *Engine) newTestQuery(f func(context.Context) error) Query {
 	return qry
 }
 
+func (ng *Engine) TestEnablePerStepStats()              { ng.enablePerStepStats = true }
```

Comment:
Is it not possible to set these via engine opts?

Reply:
Yes it is possible, but there wasn't a way to get those options in to `NewTestEngine`.
I've pushed a commit which adds 20 `false` parameters so we can do it that way.
The other way would be for this one test to not call `NewTestEngine`.